app/main.py

```diff
@@ -1,2 +1,80 @@
+from dataclasses import dataclass
+from typing import Hashable, Any, Optional
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: Any
+    hash_num: int
+
+
 class Dictionary:
-    pass
+    INITIAL_CAPACITY = 8
+    THRESHOLD = 2 / 3.0
```

Comment:
The `THRESHOLD` should be a float to ensure proper division. Consider using `2.0 / 3.0` instead of `2 / 3.0` to avoid integer division issues.